apps/auth-frontend/src/forgot-password/__tests__/ForgotPassword.test.tsx

```diff
@@ -28,15 +29,15 @@ it('renders the email in an input field', () => {
   expect(getByLabelText(/e-?mail/i)).toHaveValue('john.doe@example.com');
 });
 
-it('emits email change events', () => {
+it('emits email change events', async () => {
   const handleEmailChange = jest.fn();
   const { getByLabelText } = render(
     <ForgotPassword email="" setEmail={handleEmailChange} />,
     { wrapper: MemoryRouter },
   );
 
-  userEvent.type(getByLabelText(/e-?mail/i), 'john.doe@example.com', {
-    allAtOnce: true,
+  fireEvent.change(getByLabelText(/e-?mail/i), {
```

Comment:
I don't even know, in some places `userEvent.type` no matter with what options just broke :shrug: it fires change events with each individual character :shrug: